README.md

```diff
@@ -105,15 +105,15 @@ sudo dnf install podman
 systemctl --user start podman.service
 systemctl --user status podman.service
 systemctl --user status podman.socket
-DOCKER_HOST=unix:///run/user/$UID/podman/podman.sock poetry run pytest tests/containers --image quay.io/opendatahub/workbench-images@sha256:e98d19df346e7abb1fa3053f6d41f0d1fa9bab39e49b4cb90b510ca33452c2e4
+DOCKER_HOST=unix:///run/user/$UID/podman/podman.sock poetry run pytest tests/containers -m 'not openshift' --image quay.io/opendatahub/workbench-images@sha256:e98d19df346e7abb1fa3053f6d41f0d1fa9bab39e49b4cb90b510ca33452c2e4
```

Comment:
Do we want to have some section with possible valid marker values somewhere in our readme? :thinking:

Reply:
for sure I would not want to duplicate that, but mentioning markers in README.md is not wrong

I'm hoping to get rid of the marker again in a couple more additional PRs, when I have the deployment running on docker/openshift uniformly